### ISSUE TITLE
CS1591 - False positive for interface-implementing member

### ISSUE THREAD

**Issue by ErikWe** [dotnet/roslyn#65386]
[CS1591](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1591) - `Missing XML comment for publicly visible type or member 'Type_or_Member'`
**Version** - `MSBuild 17.4.0.51802`

### Brief description
Members that implement an interface member will implicitly apply `/// <inheritdoc/>` ([source](https://github.com/dotnet/roslyn/issues/47436#issuecomment-686827033)) - and this works as expected. However, `CS1591` is still issued for such members (unless `/// <inheritdoc/>` is explicitly added).

### Sample

Hovering `Foo.Bar` in the following sample displays `The bar.` - but `CS1591` is also issued for `Foo.Bar`.

```csharp
// in .csproj: <GenerateDocumentationFile>true</GenerateDocumentationFile>

public interface IFoo
{
    /// <summary>The bar.</summary>
    int Bar { get; }
}

public class Foo : IFoo
{
    public int Bar => 2;
}
```

**Comment by CyrusNajmabadi** [comment]
That inheritdoc behavior is purely an IDE-time thing to help with features like quick-info.  It's not a feature of the language, and docs are not inherited as far as the compiler and the rules around doc files are concerned.  So the compiler is working as intended here.
